docs/community/jobcenter.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How to Post A Job
+sidebar_position: 7
```

Comment:
i can see 1-5 are used but 6?

```suggestion
sidebar_position: 6
```

Reply:
> i can see 1-5 are used but 6?

6 is reserved for case studies instructions with PR #442  😄